xbmc/epg/EpgInfoTag.cpp

```diff
@@ -51,7 +51,7 @@ CEpgInfoTag::CEpgInfoTag(void) :
     m_iSeriesNumber(0),
     m_iEpisodeNumber(0),
     m_iEpisodePart(0),
-    m_iUniqueBroadcastID(-1),
+    m_iUniqueBroadcastID(0),
```

Comment:
Good point. How would migration work? Like changing

<pre>newTag->m_iUniqueBroadcastID = m_pDS->fv("iBroadcastUid").get_asInt();</pre>

to 

<pre>
// Compat: null-value for broadcast uid changed from numerical -1 to 0 with PVR Addon API v4.0.0
int iBroadcastUID = m_pDS->fv("iBroadcastUid").get_asInt();
newTag->m_iUniqueBroadcastID = iBroadcastUID == -1 ? PVR_TIMER_NO_EPG_UID : iBroadcastUID</pre>

here https://github.com/xbmc/xbmc/blob/master/xbmc/epg/EpgDatabase.cpp#L230? Or is it not that simple?